manifests/m/Microsoft/LAPS/6.2.0.0/Microsoft.LAPS.installer.yaml

```diff
@@ -3,11 +3,14 @@
 
 PackageIdentifier: Microsoft.LAPS
 PackageVersion: 6.2.0.0
+Scope: machine
 InstallerLocale: en-US
-InstallerType: wix
+InstallerType: msi
+InstallerSwitches:
+  Silent: /qn /norestart
+  SilentWithProgress: /qn /norestart

```

Comment:
Can you revert this? wingetcreate detected the right thing, the installer was created with WiX. winget automatically adds the correct installer switches for WiX installers.